evalml/tests/automl_tests/test_automl.py

```diff
@@ -288,7 +292,7 @@ def _dummy_callback(param1, param2, param3):
         'Allowed Pipelines': [],
         'Patience': search_params['patience'],
         'Tolerance': search_params['tolerance'],
-        'Data Splitting': 'StratifiedKFold(n_splits=5, random_state=None, shuffle=False)',
+        'Data Splitting': ('BalancedClassificationDataCVSplit(balanced_ratio=None,', 'n_splits=5, random_seed=0'),
```

Comment:
@freddyaboulton In this case, we are checking that each value in the tuple appears in the string, so `'BalancedClassificationDataCVSplit(balanced_ratio=None,'` and `'n_splits=5, random_seed=0'` are separately checked. We do this check because there's some weird spacing that happens in the output here, I think due to the formatting that these params are output in. I wasn't able to capture the spacing properly while making the string understandable, so I worked with this instead.

Reply:
Got it! Thanks for explaining that!